runelite-api/src/main/java/net/runelite/api/ScriptID.java

```diff
@@ -139,6 +139,15 @@
 	 */
 	public static final int DIARY_QUEST_UPDATE_LINECOUNT = 2523;
 
+	/**
+	 * Handles zoom input
+	 * <ul>
+	 * <li> int  zoom value </li>
```

Comment:
Both parameters being labeled as "zoom value" without further information is not helpful.